packages/gatsby-plugin-image/src/components/lazy-hydrate.tsx

```diff
@@ -1,6 +1,5 @@
 import React, { MutableRefObject } from "react"
-// @ts-ignore - react 18 has createRoot
-import { hydrate, render, createRoot } from "react-dom"
+import ReactDOM from "react-dom"
```

Comment:
Wouldn't this rather need to import namespace and not default export? (at least that's how we use it later on). Tho it works so feel free to ignore
```suggestion
import * as ReactDOM from "react-dom"
```

Reply:
It must do the same under the hood then as the warning then will show again:

![image](https://user-images.githubusercontent.com/16143594/126466144-93203b8e-4a44-469f-890f-5d4099aa069a.png)